modules/sdk-core/src/bitgo/coinFactory.ts

```diff
@@ -37,9 +37,6 @@ export class CoinFactory {
    * @throws Error
    */
   public register(name: string, coin: CoinConstructor): void {
-    if (this.coinConstructors.has(name)) {
```

Comment:
Do we run the risk of stored run time variable references being lost that may be needed by overwriting the last value in the Map?

Reply:
Not sure I follow, but if some client code has reference to an older coin and then re-registers it, the first reference they have would still be valid. It's up to the client to manage these with the modular approach, we just need to make sure we're not throwing unnecessary exceptions anymore.